src/includes/performance/concurrency/native.mdx

```diff
@@ -34,3 +34,17 @@ Following up on that warning, `sentry_transaction_context_set_name()`, which ope
  */
 
 ```
+
+Those familiar with performance monitoring on other platforms may note that scopes are one of the tools to work around concurrency. This is not the case for the native SDK, and is discussed in the following two sections. Expectations for users adding manual instrumentation to their app that are exclusive to the native SDK due to scopes are also detailed below.
```

Comment:
We write in second person unless there's a good reason to write in third person. Also the tone is a bit formal. I've changed it to second person tried to make it a bit more concise. If I've oversimplified the last bit, update as it makes sense.

```suggestion
While you can use scopes to work concurrency on other platforms, this isn't the case for the native SDK. This, as well as the what you can expect if you add manual instrumentation to your app using the native,  is discussed in the following two sections. 
```